x-pack/plugins/cases/public/components/add_comment/index.test.tsx

```diff
@@ -208,4 +212,33 @@ describe('AddComment ', () => {
       expect(wrapper.find(`[data-test-subj="add-comment"] textarea`).text()).toBe('[title](url)');
     });
   });
+
+  it('should clear session storage on submit', async () => {
```

Comment:
Do you mind if we use the React Testing Library? We try to use it on new tests. There is a utility test function called `createAppMockRenderer` which can help you render the component wrapped with our test provider. You can check other tests for examples.

Reply:
Fixed